05-JWT-Basics/starter/controllers/final/db/connect.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+const connectDB = (url) => {
+  return mongoose.connect(url, {
+    useNewUrlParser: true,
```

Comment:
You may need to delete some/all of these params to make sure the mongo connection works